package.json

```diff
@@ -20,6 +20,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "overload-protection": "^1.2.3",
```

Comment:
I feel like there is a doc about using exact versions rather than `^` but I can't find it now. Does this ring any bells for you?

Reply:
I'm all for using exact versions, overlooked since it was a c&p so good catch. 

I think there has been some whispers of using exact versions but I haven't seen or heard anything concrete.